client/widgets/test-run-state/state.directive.js

```diff
@@ -0,0 +1,184 @@
+angular.module('probedock.testRunStateWidget').directive('testRunStateWidget', function() {
```

Comment:
Applied the first solution, it is probably a better pattern for next situations like this.